### ISSUE TITLE
MSID: 85111 DOI: 2022.11.08.515698

### ISSUE THREAD

**Issue by nlisgo** [elifesciences/enhanced-preprints-import#166]
[PLACE MANUSCRIPT AND EDITOR DETAILS HERE WHEN AVAILABLE] See step 3

[PLACE PDF URL HERE WHEN AVAILABLE] See step 7

## Step 1. Inform bioRxiv

Who can help: @QueenKraken, @nlisgo, @scottaubrey

- [ ] Manuscript is in data hub index (https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v1/get-by-doi?preprint_doi=10.1101%2F2022.11.08.515698)

or (only one should be ticked. remove other from description.)

- [ ] Confirmed with Ted at bioRxiv that MECA archive is available

Send the following email to Ted and wait for his reply.

```
Hi Ted,

Please can you prepare the preprint MECA for 10.1101/2022.11.08.515698

Thanks
```

## Step 2. Create preview of manuscript

Who can help: @fred-atherden, @nlisgo, @scottaubrey

- [ ] Manuscript is available for preview (https://prod--epp.elifesciences.org/preview/10.1101/2022.11.08.515698)
    - [ ] Trigger import for staging: https://prod--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://prod-elife-epp-data/data
    - [ ] Trigger import for staging: https://staging--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://staging-elife-epp-data/data
    - [x] Pull request created to [enhanced-preprint-data](https://github.com/elifesciences/enhanced-preprints-data/pulls)
    - [x] Post link to pull request below

Pull request: https://github.com/elifesciences/enhanced-preprints-data/pull/72

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-manuscript

## Step 3: Awaiting public reviews

Who can help: Editorial team

- [x] Public reviews available on sciety (https://sciety.org/articles/activity/10.1101/2022.11.08.515698)
- [ ] Manuscript and editor details at the top of this issue (Supplied by Editorial team)

<details>
<summary>Example</summary>

```
"msas": "Genetics and Genomics", "Neuroscience"
"msid": "84628"
"version": "1"
"preprintDoi": "10.1101/2022.10.28.514241"
"articleType": "Reviewed Preprint"
"status": "Published from the original preprint after peer review and assessment by eLife."

"Reviewed Preprint posted": "2023-01-02"
"Sent for peer review": "2022-10-28"
"Posted to bioRxiv": "2022-11-21" (link: "Go to bioRxiv": "https://www.biorxiv.org/content/10.1101/2022.10.28.514241v1")

Editors:
Reviewing Editor
Michael B Eisen
University of California, Berkeley, United States
Senior Editor
Michael B Eisen
University of California, Berkeley, United States
```

</details>

## Step 4: Modify manuscripts.json (no PDF)

- [x] Reviewed preprint is published to EPP (https://elifesciences.org/reviewed-preprints/85111)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below
- [ ] Request that a doi be registered
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

Pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/684
[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

<details>
<summary>Instructions to modify manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/publish-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/334/files

Once the pull request is merged in it should be available a few minutes later.

</details>

### Request that a doi

Post the following in [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77):

```
@Fred can you register a doi for https://elifesciences.org/reviewed-preprints/85111
```

## Step 5: Awaiting search reindex

- [ ] Reviewed preprint is avaliable on journal homepage (https://elifesciences.org)

The search reindex is triggered once an hour. We need the reviewed preprint to be indexed as the search application serves the journal homepage.

<details>
<summary>Additional info</summary>

If needed, the jenkins pipeline to reindex search can be triggered sooner.

https://alfred.elifesciences.org/job/process/job/process-reindex-reviewed-preprints/

</details>

## Step 6: Published! Request PDF generation

- [ ] Post the link to the PDF url at the top of the issue
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

Post the following to the [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) on slack:

```
@Ryan Dix-Peek please can you generate a PDF for https://elifesciences.org/reviewed-preprints/85111
```

## Step 7: Introduce PDF to data folder and git repo

- [ ] PDF is avaliable at https://github.com/elifesciences/enhanced-preprints-data/raw/master/data/10.1101/2022.11.08.515698/2022.11.08.515698.pdf

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-pdf

## Step 8: Add PDF url to manuscripts.json

- [ ] Reviewed preprint PDF is available for download (https://prod--epp.elifesciences.org/reviewed-preprints/85111)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below

[PLACE LINK TO PULL REQUEST HERE]

<details>
<summary>Instructions to add PDF url to manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/add-pdf-url-to-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/397/files

Once the pull request is merged in it should be available a few minutes later.

</details>

## Step 9: Done!

- [ ] Kettle is on!

**Comment by acollings** [comment]
First REVISED Reviewed Preprint to publish

**Comment by QueenKraken** [comment]
@acollings checking on the flexibility of the publish date on the ticket. The team is in the 3 day exploration next week and will probably need Thurs & Fri to continue working on the revised preprint.

Additional question: should we expect more public reviews on Sciety, triggered by this revised version?

**Comment by acollings** [comment]
Thanks @QueenKraken yep, happy to move it, let's aim for April 24, if that's okay? Yes, we'll have an updated assessment and updated public reviews to post to Sciety. I was going to wait until a few days before publication, but please let me know if it would help to have them posted sooner!

**Comment by QueenKraken** [comment]
That's awesome, thank you :D

**Comment by QueenKraken** [comment]
@acollings can you hold back on posting public reviews to allow us to complete work this week. Then (providing we get code in this week) you can post them on Monday - would that work?

**Comment by acollings** [comment]
@QueenKraken I realise we'd forgotten to include the latest author responses, but they've been posted now. Please note that they include some figures (which display on bioRvix and hypothesis but not on Sciety): will that be okay? Also, when do you think we'll get to see a preview? Thanks!

**Comment by scottaubrey** [comment]
> Please note that they include some figures (which display on bioRvix and hypothesis but not on Sciety): will that be okay? 

@acollings I can answer this one - that shouldn't be a problem. The content we will eventually read from is here: https://sciety.org/evaluations/hypothesis:lxpxhN9cEe2uucduJPd1xg/content , which contains the figures as images. I quickly put together that content into a different paper in EPP just to see how it _should_ look:


<img width="773" alt="Screenshot 2023-04-21 at 8 28 19 am" src="https://user-images.githubusercontent.com/375812/233571323-96f307b8-637a-49a2-b015-2380d635ad1d.png">
<img width="686" alt="Screenshot 2023-04-21 at 8 28 23 am" src="https://user-images.githubusercontent.com/375812/233571334-eba2a9d7-ccdf-4509-b1c1-918425f9eb0a.png">
<img width="683" alt="Screenshot 2023-04-21 at 8 28 35 am" src="https://user-images.githubusercontent.com/375812/233571340-5b47d58c-51e9-4898-933c-8c4b5f7e0f02.png">

So not the same as a guarantee it definitely will be working when we wire up the revised version, but I can't foresee any issues at this stage with what we know.

**Comment by acollings** [comment]
@scottaubrey that's great. (If it gets very messy, we can host the figures and link out to them, but let's see how this goes.)

**Comment by QueenKraken** [comment]
Thanks @scottaubrey

**Comment by acollings** [comment]
@fred-atherden FYI, in light of feedback from Mike, I'm going to make some changes to the author response before publication. Will keep you posted.

**Comment by fred-atherden** [comment]
OK - thanks

**Comment by acollings** [comment]
@fred-atherden I've updated the author response. I can't recall if I added the blocked label or not! In any case, from my perspective, we can proceed.
